decred/decred/crypto/crypto.py

```diff
@@ -9,9 +9,9 @@
 import hashlib
 import hmac
 
-from base58 import b58decode, b58encode
-from blake256.blake256 import blake_hash
-import nacl.secret
+from base58 import b58decode, b58encode  # type: ignore
+from blake256.blake256 import blake_hash  # type: ignore
+import nacl.secret  # type: ignore
```

Comment:
While the standard library has type hints, most other libraries don't, so we're telling mypy to avoid trying to check their types.

Reply:
We could potentially play with custom stub files for these, but this is good for now. 

https://mypy.readthedocs.io/en/stable/stubs.html#stub-files